ompi/mca/sharedfp/sm/sharedfp_sm.c

```diff
@@ -105,7 +110,7 @@ struct mca_sharedfp_base_module_1_0_0_t * mca_sharedfp_sm_component_file_query(o
     char *sm_filename;
     int comm_cid = -1;
     int pid = ompi_comm_rank (comm);
-    
+
```

Comment:
Can you remove this spurious extra line (and also the extra blank line on line 129)?  It would be great to just have a patch with the `sys/stat.h` fix.  Thanks!

Reply:
I can remove that, but to be clear, the file currently has trailing whitespace introduced by someone else in the past and my text editor automatically trimmed that.

Reply:
No worries.  Don't get me wrong; I'm all in favor of fixing up whitespace stuff like that, but we generally prefer to put those kind of changes in separate separate commits.